Hackney.Core/NuGet.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<config>
+		<add key="github" value="https://nuget.pkg.github.com/LBHackney-IT/index.json" />
+	</config>
+	<packageSources>
+        <clear />
+	</packageSources>
+    <packageSourceCredentials>
+        <github>
+            <add key="Username" value="humulla" />
+            <add key="ClearTextPassword" value= "%NUGET_KEY%" />
+        </github>
+    </packageSourceCredentials>
```

Comment:
Having this checked in (esp. with a specific non-system users username) seems wrong. Can it be removed?